pkg/catalog/outbound_traffic_policies.go

```diff
@@ -135,6 +150,36 @@ func (mc *MeshCatalog) ListAllowedOutboundServicesForIdentity(serviceIdentity id
 	return allowedServices
 }
 
+func (mc *MeshCatalog) buildMultiClusterGatewayPolicies() []*trafficpolicy.OutboundTrafficPolicy {
+	var outPolicies []*trafficpolicy.OutboundTrafficPolicy
+
+	k8sServices := mc.kubeController.ListServices()
+	var destServices []service.MeshService
+	for _, k8sService := range k8sServices {
+		destServices = append(destServices, utils.K8sSvcToMeshSvc(k8sService))
+	}
+
+	for _, destService := range destServices {
+		if !destService.Local() {
+			continue
+		}
+		hostnames, err := mc.GetServiceHostnames(destService, service.RemoteCluster)
+		if err != nil {
+			log.Error().Err(err).Msgf("Error getting service hostnames for service %s", destService)
+			continue
+		}
+
+		weightedCluster := getDefaultWeightedClusterForService(destService)
+		policy := trafficpolicy.NewOutboundTrafficPolicy(destService.FQDN(), hostnames)
+		if err := policy.AddRoute(trafficpolicy.WildCardRouteMatch, weightedCluster); err != nil {
+			log.Error().Err(err).Msgf("Error adding route to outbound policy in permissive mode for destination %s(%s)", destService.Name, destService.Namespace)
+			continue
+		}
+		outPolicies = append(outPolicies, policy)
```

Comment:
Why is an RDS route required for SNI passthrough clusters?

Reply:
oops, that was an old change, removed now.